index.bs

```diff
@@ -8058,6 +7765,79 @@ for an interface is to be implemented.
 </div>
 
 
+<h4 id="Default" extended-attribute lt="Default">[Default]</h4>
+
+If the [{{Default}}] [=extended attribute=] appears on a [=regular operation=],
+then it indicates that steps described in the [=corresponding default operation=]
+must be carried out when the operation is invoked.
+
+The [{{Default}}] extended attribute must
+[=takes no arguments|take no arguments=].
+
+The [{{Default}}] extended attribute must not
+be used on anything other than a [=regular operation=]
+for which a [=corresponding default operation=] has been defined.
+
+<div class="example">
+
+    As an example, the [{{Default}}] extended attribute is suitable
+    for use on the “toJSON” [=regular operation=] of the
+    <a href="https://w3c.github.io/browser-payment-api/#paymentresponse-interface"><code class="idl">PaymentResponse</code></a>
+    interface [[payment-request]].
+
+    <pre highlight="webidl">
+        [SecureContext]
+        interface PaymentResponse {
+          readonly attribute DOMString paymentRequestId;
```

Comment:
Note that this is probably changing to an `id` member in payment request. Maybe using a made-up example is better.